ktor-server/ktor-server-tomcat/jvm/test/io/ktor/tests/server/tomcat/TomcatEngineTest.kt

```diff
@@ -57,4 +57,10 @@ class TomcatSustainabilityTestSuite :
         }
         enableHttp2 = false
     }
+
+    @Ignore
```

Comment:
Why do we ignore in the case of Tomcat?
Maybe add a comment?

Reply:
Thanks for the notice